inst/roxygen2-tags.yml

```diff
@@ -350,20 +388,23 @@
     Add an arbitrary section to the documentation. Now generally superseded
     in favour of using a level 1 heading.
   template: " {section title}:\n"
+  snippet: '${1:section title}'
   vignette: rd-formatting
 
 - name: seealso
   description: >
     Link to other related functions or urls. Usually a sentence or two, or
     a bulleted list.
   template: ' {[func1()], <url1>}'
+  snippet: '[${1:func}()]'
```

Comment:
Not sure what to do with `<url>`. Maybe we could have another `seealso`, i.e. 

```
name: seealso
   description: >
     Link to a url. Usually a sentence or two, or a bulleted list.
   snippet: '<${1:url}>'
``` 

so that the user could choose.

Reply:
I think it's fine if the snippet doesn't cover every case.